metricbeat/docs/fields.asciidoc

```diff
@@ -1192,6 +1192,83 @@ type: long
 Last updated
 
 
+[float]
+== osd_df fields
```

Comment:
One thing that popped up in my head is that we have here `df` and we have `system.filesystem` with similar info. Don't know if `osd_filesystem` would make sense here (even though pretty long).

Reply:
Actually the function of  *osd df* in ceph is very similar with *system.filesystem * in linux. It shows osds' weight, reweight, size, used, avai and used pct information. Naming this  new metricbet as *osd_df* is because it's a standard command in Ceph, and I consider that it will be much more familer to ceph users. For more about the command's detials:
http://docs.ceph.com/docs/master/man/8/ceph/

Reply:
We had a similar discussion with the system module. Most sysadmins will be used to run the `df` command. But someone consuming the data is more looking for information about the filesystem which could be a different person. TBH don't know which one is better. I'm fine moving forward with `df`.